### DISCUSSION TITLE
Talkgroup Range with SDR Trunk?

### DISCUSSION THREAD

**Discussion by DMS11X** [chuot/rdio-scanner#149] [Q&A]
Getting this RDIO setup with my SDRTrunk on a dedicated Windows PC.

Having a bit of a struggle as the PDF instructions aren't exactly clear for configuring with SDRTrunk...

However my question is, how do I setup a NewTalkgroup as a "range" of ID's?

My setup is monitoring two different new Phase 2 TDMA CQPSK sites, and I have setup the SDRTrunk to monitor all talkgroups...

**Comment by chuot** [comment]
Rdio Scanner works its data from MP3 tags generated by SDR Trunk. If you create a range for multiple talkgroups, they will be picked up by the dirWatch (type=sdrtrunk) and stored in the appropriate system/talkgroup. The system label in Rdio Scanner must match the one defined in the SDR trunk for the system. If it doesn't exist, Rdio Scanner will create a new one, as long as the auto populate option is enabled, which is the default. The only problem I see is that all the buttons in Rdio Scanner will have the same label, the one given to the alias range. You will need to change the talkgroup label each time on Rdio Scanner when a new talkgroup is auto populated. That's why it's better to have one recording alias per talkgroup so that the Rdio Scanner gets a different label per talkgroup. It will also be automatically updated in Rdio Scanner if you change the alias name on SDR Trunk. Again, this auto populate feature will only work if enabled in Rdio Scanner.

**Reply by DMS11X** [reply]
Thank you so very much sir, very thorough response, truly appreciated. By the way, do you know of any youtube tutorial / setup videos for RDIO, I tried searching and couldn't really find any.

**Reply by chuot** [reply]
Not that I know of. sorry.
